pfcpiface/messages.go

```diff
@@ -73,6 +73,7 @@ func (pc *PFCPConn) handleAssociationSetupRequest(upf *upf, msg message.Message,
 	// Timestamp shouldn't be the time message is sent in the real deployment but anyway :D
 	log.Println("Dnn info : ", upf.dnn)
 
+	networkInstance := encodeFQDN(upf.dnn)
```

Comment:
```suggestion
	networkInstance :=string(ie.NewNetworkInstanceFQDN(up4.dnn).Payload)
```

Reply:
is ie.NewNetworkInstanceFQDN available in go-pfcp ?

Reply:
Yup.
https://pkg.go.dev/github.com/wmnsk/go-pfcp/ie#NewNetworkInstanceFQDN